BuildNotifications/ViewModel/Tree/BuildNodeViewModel.cs

```diff
@@ -134,22 +146,32 @@ public double ProgressToDisplay
 
         public string StatusDisplayName => StringLocalizer.Instance[_buildStatus.ToString()];
 
-        public BuildReason Reason => Node.Build.Reason;
-
         public int UserColumns => RequestedByIsSameAsFor ? 1 : 2;
 
         public override void BackendPropertiesChanged()
         {
             BackendPropertiesChangedInternal();
         }
 
-        protected override BuildStatus CalculateBuildStatus() => _buildStatus;
+        protected override BuildStatus CalculateBuildStatus()
+        {
+            return _buildStatus;
+        }
 
-        protected override DateTime CalculateChangedDate() => _changedDate;
+        protected override DateTime CalculateChangedDate()
+        {
+            return _changedDate;
```

Comment:
Why change this? Wasn't the expression body better readable?

Reply:
Good question. Seems like my Code Cleanup profile was not configured properly. Fixed.